build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
```

Comment:
Please move build tool and gradle updates into a separate commit.
This keeps the commits atomic.